.github/workflows/validate-nginx-config.yml

```
@@ -0,0 +1,63 @@
+        type: string
+      templatedir: # Ref: https://github.com/nginxinc/docker-nginx-unprivileged/blob/main/entrypoint/20-envsubst-on-templates.sh#L32
+        description: 'Nginx configuration template directory. Usually have files with .template extension.'
+        required: true
+        type: string
+      verbose:
+        default: true
+        description: 'Toggle verbose output. By setting this to true, the job will print out the nginx version, nginx config files, and generated config files.'
+        required: false
+        type: boolean
+      workdir:
+        default: 'web'
```

actionlint (1.7.4)

input "workdir" of workflow_call event has the default value "web", but it is also required. if an input is marked as required, its default value will never be used

(events)